projects/igniteui-angular/src/lib/directives/for-of/virtual.helper.component.ts

```diff
@@ -9,28 +9,27 @@ import { PlatformUtil } from '../../core/utils';
     template: '<div #container class="igx-vhelper__placeholder-content" [style.height.px]="size"></div>',
     standalone: true
 })
-export class VirtualHelperComponent extends VirtualHelperBaseDirective implements OnInit, OnDestroy  {
+export class VirtualHelperComponent extends VirtualHelperBaseDirective implements OnDestroy  {
     @HostBinding('scrollTop')
     public scrollTop;
 
-    @HostBinding('style.width.px')
-    public scrollWidth;
```

Comment:
Hmm, looking at it, this is now a breaking change. This was a public property and you're removing it.